core/domain/stats_domain.py

```diff
@@ -806,8 +981,15 @@ class Playthrough:
     """Domain object representing a learner playthrough."""
 
     def __init__(
-            self, exp_id, exp_version, issue_type, issue_customization_args,
-            actions):
+        self,
+        exp_id: str,
+        exp_version: int,
+        issue_type: str,
+        issue_customization_args: Dict[str, Union[str, int, Dict[
+            Union[str, int], Union[str, int]
```

Comment:
Ahh, I think @winnie368c you should change this type. Because looking at the test cases I think it should be `Dict[str, Dict[str, Union[str, int, ....]]` or something similar. For more info about `customization_args` you can read here. https://github.com/oppia/oppia/blob/2c89393c66636b399b93dc1397486c2f45e577f3/core/domain/customization_args_util.py#L84

Reply:
Also, MyPy threw some errors like `unused 'type: ignore' comment` 
This means you should remove those ignores.

Reply:
Apart from this everything looks good to me!!